website/content/docs/usage/notebook-sample/index.md

```diff
@@ -1,99 +1,172 @@
 ---
-title: Notebook sample
+title: Kubeflow notebook sample
 weight: 10
 ---
 
-{{< warning >}}
-Incomplete. Show usage of OPA, usage of Egeria, Starting the notebook, Writing code; Also should be a KubeFlow Notebook sample.
-{{</ warning >}}
+This sample shows how to run a Kubeflow notebook with {{< name >}} and demonstrate how polices are seamlessly applied when accessing a dataset.
 
-This guides describes sample usage of {{< name >}} by describing a sample Jupyter notebook application.
-It describes how you, as a data scientist, can use a Jupyter notebook for building a fraud detection model using a 
-data asset you find in the data catalog. 
+## Before you begin
+This sample guide assume that you have completed all the steps in [quick start guide](/docs/setup/quickstart/).
+
+Ensure that you have the following:
+- Kubernetes cluster (this guide was tested with kind v0.10.0 and OpenShift 4.3)
```

Comment:
This + kubectl are redundant because of the link to quick start. Consider removing and if not, using the same syntax/text as in the quick start

Reply:
prefer to keep. updated the text.